templates/index.html

```diff
@@ -36,8 +36,15 @@
         {% if request.path == '/new_intake' %}
         <a style="background: rgb(73, 73, 223);" href="/assigned_projects">Allocate Students to Projects</a>
         {% endif %}
-
-
+        {% if request.path == '/outstanding' %}
+        <!--
+        <a target="_blank" href="mailto:?subject=Acknowledgement of your application to the WEHI student internship program&body=Hi All,%0D%0A%0D%0AThank you for applying for the WEHI Student Internship program.%0D%0A%0D%0A
+I will be calling you by mobile phone over the next week to get a better understanding of projects that fit your interests and career aspirations. Please let me know if you need to make other arrangements as I can be flexible.%0D%0A%0D%0A
+You can see a list of projects as well as other details on the WEHI Student Internship page, where you can click on the project to go to a project page.%0D%0A%0D%0AEach project page has a rough description of the role outline, but the key information is the 'Further Reading' section that may be available for that project.%0D%0A%0D%0A
+Please also be acquainted with the program handbook.%0D%0A%0D%0ADetails of duties are available via the student projects page at https://wehi-researchcomputing.github.io/students.%0D%0A%0D%0ADetails of the intake dates are available at https://wehi-researchcomputing.github.io/intake_dates.%0D%0A%0D%0AA FAQ is also available at https://wehi-researchcomputing.github.io/faq">Copy Email Acknowledgement</a>
+        -->
+        <a href="#" id="emailAcknowledgementBtn">Email Acknowledgement</a>
+        {% endif %}
 {% endblock %}
```

Comment:
probably good idea to move the anchor tag button within /new_intake path as the button will only show up when a user enters list of new intake students table